### ISSUE TITLE
Net value incorrect

### ISSUE THREAD

**Issue by altitdb** [altitdb/taxidobarba#85]
In anual summary report, the net value is incorrect when doesn't have spents. Net and gross value must be equals.

**Comment by fernandoWPF** [comment]
As long as there is travel, there will be a difference between the netvalue and the grossvalue, because to calculate the net value of the trip, we make the price of the trip - amount paid to the driver

**Comment by altitdb** [comment]
Yes, but, I need that in all places, I have gross value like all gains, like: gross value = city value + (price of the trip - amount paid to the driver), and, like: net value = city value + (price of the trip - amount paid to the driver) - other spents (gas, tickets, etc).
